pkg/collector/tcp.go

```diff
@@ -49,24 +49,20 @@ func (cp *CollectingProcess) startTCPServer() {
 					return
 				}
 			}
-			go cp.handleTCPClient(conn, stopCh)
+			go cp.handleTCPClient(conn)
 		}
 	}(cp.stopChan)
 	<-cp.stopChan
```

Comment:
Don't you want to include `CloseAllClients` method here?

Reply:
`CloseAllClients` is put in the `Stop()` function